launch/zed.launch

```diff
@@ -2,6 +2,9 @@
     <arg name="run_perception" default="true"/>
     <arg name="run_visual_odom" default="false"/>
 
+    <arg name="rvizconfig" default="$(find mrover)/config/rviz/auton_sim.rviz" />
```

Comment:
I don't think we need to add this here since this will only be run the jetson which isn't connected to a monitor (rviz is run from the basestation).